src/test/regress/expected/multi_behavioral_analytics_basics.out

```diff
@@ -225,16 +222,15 @@ TRUNCATE agg_results;
 INSERT INTO agg_results(user_id)
 SELECT user_id
 FROM users_table
-WHERE (value_1 = 10
-       OR value_1 = 11
-       OR value_1 = 12)
+WHERE (value_1 = 1
+       OR value_1 = 2)
 GROUP BY user_id
```

Comment:
This query has a meaning of having at least 2 out of 3, so could you also keep third OR filter?